### ISSUE TITLE
frequency bug

### ISSUE THREAD

**Issue by pnrobinson** [pnrobinson/hphenote#50]
When the user adds a Freqeuncy Ontology term, e.g., 
https://hpo.jax.org/app/browse/term/HP:0040281,
then the label (Very frequent) gets added instead of the HPO term id (HP:0040281).
This is not the correct format and causes errors in downstream pipelines with phenol.

**Comment by kingmanzhang** [comment]
We can change it to term id when saving files. Let's talk when you are back. I have some pending issues to discuss with you.

**Comment by kingmanzhang** [comment]
It now uses id if frequency is from the dropdown choice box.
